gold-digger/src/presenters/homePresenter.js

```diff
@@ -1,45 +1,48 @@
-
-import HomeView from "../views/homeView.js";
-import {redirect} from "react-router-dom";
-import { useEffect } from "react";
-import PrevlistView from "../views/prevListView.js";
-
-
-function Home(props) {
-    useEffect(onMountedACB, [props.model]);
-
-    return (
-      <HomeView generatePlaylist={generatePlaylistACB}></HomeView>  
-    );
-
-    /*
-
-    Lifecycle: GET prev playlists from Firebase
-    For each previous playlist Event: onClick inspect playlist
-
-    */
-    function onMountedACB() {
-        console.log("access: " + localStorage.getItem("access-token"));
-
-        if (localStorage.getItem("access-token") === null && localStorage.getItem("log-in") === "true") {
-            localStorage.setItem("access-token", "in progress");
-            props.model.requestToken();
-        } else {
-            //props.model.logout();
-        }
-
-        // ACB to run at the end
-        return function onUnmountedACB() {
-        }
-    }
-    
-    
-    
-    /* Event: Set window location to Source */
-    function generatePlaylistACB() {
-        // go to next window, ie Source Presenter
-        return redirect("/source"); 
-    }
-
-}
+import HomeView from "../views/homeView.js";
+import {redirect} from "react-router-dom";
+import { useEffect } from "react";
+import PrevlistView from "../views/prevListView.js";
+
+function Home(props) {
+    useEffect(onMountedACB, [props.model]);
+
+    return (
+      <HomeView generatePlaylist={generatePlaylistACB}></HomeView>  
+    );
+
+    /*
+
+    Lifecycle: GET prev playlists from Firebase
+    For each previous playlist Event: onClick inspect playlist
+
+    */
+    function onMountedACB() {
+        console.log("HOME access token: " + localStorage.getItem("access-token"));
```

Comment:
Not entiraly sure how necessary both localstorage and model.state is, but clean up would be good to make this code more readable.

Reply:
We need model.state in order to use useEffect, and we need localstorage in order to set model.state if we lose connection. I don't see a way to get rid of either of them in the case of isLoggedIn.

As for clean-up I can remove the logs from the commit, is there anything else you are thinking of?